docs/modules/user-guide/partials/con_using-top-level-attribute.adoc

```diff
@@ -0,0 +1,38 @@
+[id="con_using-top-level-attribute_{context}"]
+= Using top-level attribute
+
+[role="_abstract"]
+
+This section explains how to use top-level attribute in devfiles in order to reuse global attributes in different sections of the same devfile.
```

Comment:
IBM SG wants us to avoid self referential language like "this section" or "this guide". 

Make it user focused, a minimalism principle and the SG wants us to be user focused as much as possible :) 

What do you think of this rephrase?: 

`Use top-level attributes in devfiles to reuse global attributes in different sections of the same devfile.`